ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -457,6 +474,17 @@ export default defineComponent({
       await parseMhrHomeOwners(currentOwnerGroups)
     }
 
+    const parseMhrHomeDetails = async (homeDetails: MhrRegistrationDescriptionIF): Promise<void> => {
+      for (const [key, value] of Object.entries(homeDetails)) {
+        setMhrHomeDescription({ key: key, value: value })
+      }
+
+      setMhrHomeDescription({
+        key: 'certificationOption',
+        value: homeDetails.csaNumber ? HomeCertificationOptions.CSA : HomeCertificationOptions.ENGINEER_INSPECTION
```

Comment:
If `description` is not there, and therefore `homeDetails`, then the value will be incorrectly set as Eng Ins.

Reply:
True! 
Should 'never' happen but yea. 
If it does, it would all fallback to '(Not Entered)' in the review component